Tests/SonarScanner.MSBuild.PreProcessor.Test/JreResolution/JreCacheTests.cs

```diff
@@ -978,11 +948,11 @@ The JRE provisioning is a time consuming operation.
                 If you already have a compatible java version installed, please add either the parameter "/d:sonar.scanner.skipJreProvisioning=true" or "/d:sonar.scanner.javaExePath=<PATH>".
                 """);
             testLogger.DebugMessages.Should().SatisfyRespectively(
-                x => x.Should().Be(@$"Starting the Java Runtime Environment download."),
-                x => x.Should().Be(@$"The checksum of the downloaded file is '347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b' and the expected checksum is '347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b'."),
-                x => x.Should().Match(@$"Starting extracting the Java runtime environment from archive '{home}\cache\347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b\OpenJDK17U-jre_x64_windows_hotspot_17.0.11_9.tar.gz' to folder '{home}\cache\347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b\*'."),
-                x => x.Should().Match(@$"Moving extracted Java runtime environment from '{home}\cache\347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b\*' to '{home}\cache\347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b\OpenJDK17U-jre_x64_windows_hotspot_17.0.11_9.tar.gz_extracted'."),
-                x => x.Should().Be(@$"The Java runtime environment was successfully added to '{home}\cache\347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b\OpenJDK17U-jre_x64_windows_hotspot_17.0.11_9.tar.gz_extracted'."));
+                x => x.Should().Be("Starting the Java Runtime Environment download."),
+                x => x.Should().Be($"""The checksum of the downloaded file is '347f62ce8b0aadffd19736a189b4b79fad87a83cc36ec1273081629c9cb06d3b' and the expected checksum is '{sha}'."""),
```

Comment:
Why not using the sha in both place here?
```suggestion
                x => x.Should().Be($"""The checksum of the downloaded file is '{sha}' and the expected checksum is '{sha}'."""),
```

Reply:
Why using a raw string literal here instead of a simple interpolated string?
```suggestion
                x => x.Should().Be($"The checksum of the downloaded file is '{sha}' and the expected checksum is '{sha}'."),
```